src/com/moclojer/server.clj

```diff
@@ -73,20 +126,43 @@
              :port http-port
              :url (str "http://" http-host ":" http-port)
              :version config/version)
-    (->
-     *router
-     (build-config-map  {:http-host http-host
-                         :http-port http-port
-                         :join? join?})
-     get-interceptors
-     http/create-server
-     http-start)))
+    (if swagger?
+      (let [router (reitit-router *router)]
+        (-> {:env                     config/moclojer-environment
+             ::http/request-logger    log/request
+             ::http/routes            []
+             ::http/type              :jetty
+             ::http/join?              join?
+             ::http/container-options {:h2c?                 true
+                                       :context-configurator context-configurator}
+ ;; allow serving the swagger-ui styles & scripts from self
+             ::http/secure-headers {:content-security-policy-settings
+                                    {:default-src "'self'"
+                                     :style-src "'self' 'unsafe-inline'"
+                                     :script-src "'self' 'unsafe-inline'"}}
+             ::http/host              http-host
+             ::http/port              http-port}
+
+            (http/default-interceptors)
+            (pedestal/replace-last-interceptor router)
+
+            (http/dev-interceptors)
+            (http/create-server)
+            http-start))
+
+      (->
+       *router
+       (build-config-map  {:http-host http-host
+                           :http-port http-port
+                           :join? join?})
+       get-interceptors
+       http/create-server
+       http-start))))
```

Comment:
this will be refactored later, but just shows that retiti and pedestal are different types of routes

Reply:
I wouldn't keep the `﻿pedestal` in this case, I would switch 100% to ﻿`retiti` _(even with ﻿swagger ﻿false)_.
For me, it doesn't make sense to keep both!